crates/cli/src/subcommands/call.rs

```diff
@@ -1,18 +1,19 @@
+use crate::api::ClientApi;
 use crate::common_args;
 use crate::config::Config;
 use crate::edit_distance::{edit_distance, find_best_match_for_name};
-use crate::util::{self, UNSTABLE_WARNING};
-use crate::util::{add_auth_header_opt, database_identity, get_auth_header};
+use crate::util::UNSTABLE_WARNING;
 use anyhow::{bail, Context, Error};
 use clap::{Arg, ArgMatches};
-use itertools::Either;
-use serde_json::Value;
+use convert_case::{Case, Casing};
+use itertools::Itertools;
 use spacetimedb::Identity;
-use spacetimedb_lib::de::serde::deserialize_from;
-use spacetimedb_lib::sats::{AlgebraicType, AlgebraicTypeRef, Typespace};
+use spacetimedb_lib::sats::{self, AlgebraicType, Typespace};
 use spacetimedb_lib::ProductTypeElement;
+use spacetimedb_schema::def::{ModuleDef, ReducerDef};
 use std::fmt::Write;
-use std::iter;
+
+use super::sql::parse_req;
```

Comment:
Why is `parse_req` in the `sql` module? Also what does `parse_req` mean. I realize that this is a pre-existing function.